autoload/ctrlp.vim

```diff
@@ -218,6 +218,7 @@ fu! s:opts(...)
 	for each in ['byfname', 'regexp'] | if exists(each)
 		let s:{each} = {each}
 	en | endfo
+	if !exists('g:ctrlp_tilde_homedir') | let g:ctrlp_tilde_homedir = 0 | en
```

Comment:
Yeah I thought so, but it is never used in the file itself. In contrast to all those other variables. 

And surprisingly the arraybonly sets default values for the s:variables.